Makefile

```diff
@@ -0,0 +1,20 @@
+deploy-localstack:
```

Comment:
Consider adding an install target and/or provide some instructions how to test the deployment